templates/rbac/aggregated-role.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubediag-view
+  labels:
+    app: {{ include "kubediag-helm.name" . }}
+    app.kubernetes.io/name: {{ include "kubediag-helm.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
These labels should be deleted.

Reply:
This role `kubediag-view` do not have `admin` nor `edit` right, so I think these labels should be deleted.